packages/enzyme-matchers/src/index.js

```diff
@@ -30,6 +33,9 @@ const assertions = {
   toBeChecked,
   toBeDisabled,
   toBeEmptyRender,
+  toContainMatch,
+  toContainMatches,
+  toContainExactlyOneMatch,
```

Comment:
we need to add these matchers to the typescript spec here: https://github.com/FormidableLabs/enzyme-matchers/blob/master/packages/jest-enzyme/src/index.d.ts

Reply:
Done